src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -378,7 +378,6 @@ public partial class SocketException : System.Exception
     {
         public SocketException() { }
         public SocketException(int errorCode) { }
-        public override string Message { get { return default(string); } }
```

Comment:
If the contract were stable, I believe we would not be able to do this.  But since it's not stable, seems like an unnecessary thing to include.  I can revert this if you'd prefer: the implementation will just end up being:
```C#
public override string Message => base.Message;
```